torch/sparse/__init__.py

```diff
@@ -1,53 +1,118 @@
 import torch
 from torch import _C
-
-import sys
+from .._utils import _type, _cuda
 
 _sparse_tensor_classes = set()
 
 
-class DoubleTensor(_C.SparseDoubleTensorBase):
+class _SparseTensorBase(object):
```

Comment:
I guess I could, though I'll probably have to override some of the base methods that aren't supported for sparse tensors (and raise `NotImplementedError` or whatever makes sense)